update-cmdlet-documentation.ps1

```diff
@@ -70,7 +80,32 @@ if ($incompleteFiles) {
     Write-Warning "Run this script again once these files have been updated in order to generate the XML help documentation for the module."
 }
 else {
-    New-ExternalHelp -Path $documentationPath -OutputPath "$PSScriptRoot/src/Chocolatey.PowerShell"
+    New-ExternalHelp -Path $documentationPath -OutputPath "$PSScriptRoot/src/Chocolatey.PowerShell" -Force
+
+    $newOrUpdatedFiles = $newOrUpdatedFiles |
+        Rename-Item -NewName { $_.BaseName + ".mdx" } -PassThru |
+        ForEach-Object {
+            $content = Get-Content -Path $_.FullName
+            $content = $content -replace '\[(?<name>[^\]]+)\]\((?<xref>[^,]+),(?<classes>[^)]+)\)', '<Xref title="${name}" value="${xref}" classes="${classes}" />'

```

Comment:
@vexx32 Can this be fixed up to also include a `anchor=""` field? The anchor would be the hash part (if one is included), so in a typical xref like this:

`[v1.1.0](xref:choco-release-notes#march-30-2022)`

Would become:

`<Xref title="v1.1.0" value="choco-release-notes" anchor="march-30-2022" />`

Reply:
sure thing :3

Reply:
Awesome, thank you :)